Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [package]
+rust-version = "1.59"
```

Comment:
humm, I'm not sure about this, is there a specific version to pin this?

Reply:
Sorry again this is accidental. In trying to debug with that segfault I updated to 1.70 which seems to be totally broken